image-upload

```diff
@@ -42,10 +42,7 @@ def upload(store, source, public, prune=False):
         return False
 
     # Start building the command
-    cmd = ["curl", "--no-progress-meter", "--fail", "--upload-file", source]
-
-    if ca := get_host_ca(url.netloc):
-        cmd += ['--cacert', ca]
+    cmd = ["curl", "--no-progress-meter", "--fail", "--upload-file", source] + get_curl_ca_arg(url.netloc)
```

Comment:
I like this syntax: `["curl", ..., *get_curl_ca_arg(url.netloc)]`

Reply:
Sure :-) Done